helmfile.d/10-managementportal.yaml

```diff
@@ -29,6 +29,12 @@ releases:
         values: [{{ .Values.server_name }}]
       - name: ingress_rate_limited.hosts
         values: [{{ .Values.server_name }}]
+      # --- Keep in secrets in sync with hostnames (main and subdomains have dedicated secrets)
+      - name: ingress.tls.secretName
+        value: radar-base-tls
```

Comment:
We used to have the same secret name for TLS but then the cert-manager logs would constantly complain about reusing a secret name. Is this not the case anymore?